libraries/AP_Mount/AP_Mount_Siyi.h

```diff
@@ -296,6 +296,8 @@ class AP_Mount_Siyi : public AP_Mount_Backend
         const char* model_name;
     };
     static const HWInfo hardware_lookup_table[];
+
+    const float NaN = nanf("0x4152");               // NaN value used for invalid data
```

Comment:
```suggestion
    static const float NaN = nanf("0x4152");               // NaN value used for invalid data
```

Reply:
I agree to this code change.
As is, if you commit, the indentation of comments will be broken.
Please match the indentation of the comments, with the others.

Reply:
If I make this suggestion I generally expect the author of the PR to do it, not bounce it back at me to do :-)